bin/main.rb

```diff
@@ -1,109 +1,6 @@
 #!/usr/bin/env ruby
-class Game
-  attr_accessor :tic_tac_toe, :player1, :player2
+require_relative '../lib/game.rb'
 
-  def initialize
-    @tic_tac_toe = Board.new
-    puts %(First player; please type your name!)
-    @player1 = Player.new(gets.chomp, 'X')
-    puts %(Second player; please type your name!)
-    @player2 = Player.new(gets.chomp, 'O')
-  end
-
-  def welcome
-    puts %(Welcome on board #{@player1.name} and #{@player2.name}, let the Tic-Tac-Toe game begins!)
-    puts %(Player 1: #{@player1.name} has token: #{@player1.token})
-    puts %(Player 2: #{@player2.name} has token: #{@player2.token})
-  end
-
```

Comment:
- [ ] Kindly remove all `puts` statements here. You should handle the `puts` and user interaction in the main.rb file. One of the possible ways to go about this is to make your methods accept one or two parameters and when you call the methods in the main.rb, it takes in the user's input as arguments and with that, you can use the value to compute your logic and then print your desired string based on the input from the user. 

   - With this, you can get to keep your code DRY and not repeating them as you have in this code.

Another way possible is to return strings instead of printing them just like the previous reviewer @majovanilla  mentioned.